src/drivers/uavcannode/UavcanNode.cpp

```diff
@@ -232,21 +234,24 @@ int UavcanNode::start(uavcan::NodeID node_id, uint32_t bitrate)
 
 		_can = new CanInitHelper();
 
-		if (_can == nullptr) {                    // We don't have exceptions so bad_alloc cannot be thrown
+		if (_can ==
```

Comment:
Can you revert these whitespace changes?

Reply:
Done